server/remote/gitea/helper.go

```diff
@@ -229,3 +230,11 @@ func matchingHooks(hooks []*gitea.Hook, rawurl string) *gitea.Hook {
 	}
 	return nil
 }
+
+func convertLabels(from []*gitea.Label) []string {
+	var labels []string
```

Comment:
```suggestion
	labels := make([]string, len(from))
```
we already know how much mem we have to allocate